File: 0771-jewels-and-stones/0771-jewels-and-stones.c
int numJewelsInStones(char* jewels, char* stones) {
    int result = 0;
    for (int i=0;i<strlen(jewels);i++) {
        for (int j=0;j<strlen(stones);j++) {
            if (jewels[i] == stones[j]) result += 1;
        }
    }
    return result;
}

File: 0771-jewels-and-stones/0771-jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        ans = {}
        result = 0
        for s in stones:
            if s not in ans.keys():
                ans[s] = 1
            else:
                ans[s] += 1
        for j in jewels:
            if j in ans.keys():
                result += ans[j]
        return result